packages/adobe-sign/src/models/agreement.ts

```diff
@@ -1,12 +1,26 @@
-export class Agreement {
-    displayDate: string;
-    esign: boolean;
-    groupId: string;
-    hidden: boolean;
-    latestVersionId: string;
-    name: string;
+export interface ResponseData {
     id: string;
-    parentId: string;
+    name: string;
+    groupId: string;
+    type: string;
     status: string;
+    hasFormFieldData: boolean;
+}
+
+export class Agreement {
+    id: string;
+    name: string;
+    groupId: string;
     type: string;
+    status: string;
+    hasFormFieldData: boolean;
+
+    constructor({id, name, groupId, type, status, hasFormFieldData}: ResponseData) {
```

Comment:
Do you have to spell out each member of ResponseData? Or can you replace those with `data: ResponseData` and reference them in the constructor with `data.id`, `data.name`, etc. I just don't like having the properties spelled out three times.